messages/en/vtex.admin-orders@0.x.json

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
Should it have a CHANGELOG? I think you're going to release a version, am I right?